### PR TITLE
[Balance] Adjustments to T12 2P based on PTR data

### PR DESCRIPTION
**Pull request by stn-ms**

### CODE REVIEW

File: sim/druid/burning_treant.go [hunk 1]
```diff
@@ -16,10 +16,16 @@ type BurningTreant struct {
 }
 
 func (druid *Druid) NewBurningTreant() *BurningTreant {
-	var burningTreantBaseStats = stats.Stats{stats.SpellCritPercent: 5}
+	baseStats := stats.Stats{stats.SpellCritPercent: 0}
```
Comment (1337LutZ) [inline sim/druid/burning_treant.go:19]: Ah yea, I guess it's the same findings I had with Fiery Imp from Warlock 2P

Comment (stn-ms) [reply sim/druid/burning_treant.go:19]: Out of all the data we gathered, there isn't a single log where fireseed has crit.

Comment (1337LutZ) [reply sim/druid/burning_treant.go:19]: Was this tested without the 5% raid debuff?

Comment (1337LutZ) [reply sim/druid/burning_treant.go:19]: Was just double checking. both Warlock and Mage show 0 base crit and 0 inherited crit 👍

**Pull request merged**